inspirehep/base/templates/format/record/Inspire_Default_HTML_brief.tpl

```diff
@@ -19,98 +19,112 @@
 
 {% from "format/record/Inspire_Default_HTML_general_macros.tpl" import render_record_authors, record_abstract, record_arxiv, record_cite_modal with context %}
 
-{% from "format/record/Inspire_Default_HTML_brief_macros.tpl" import render_doi, record_journal_info with context %}
+{% from "format/record/Inspire_Default_HTML_brief_macros.tpl" import record_journal_info, render_doi, record_journal_info_and_doi with context %}
 
 {% block record_header %}
 <div class="row">
   <div class="col-md-12">
     <div id="panel-default-brief" class="panel panel-default" >
-    <div class="panel-body" >
-      <div class="row">
-      <div class="col-md-9"  id="left-column">
-        <h4 class="custom-h">
-          <span id='checkbox-parent'>
-            <input type="checkbox" class="checkbox-results" id="{{ record['control_number'] }}">
-          </span>
-          <b>
-            <a class="title" href="{{ url_for('record.metadata', recid=record['control_number']) }}">
-              {{ record['titles[0].title']|capitalize }}
-            </a>
-            {% if record['titles[0].title']|count_words() > 5 %}
-              <a class="mobile-title" href="{{ url_for('record.metadata', recid=record['control_number']) }}">
-                {{ record['titles[0].title']|capitalize | words(5) + "..."}}
-              </a>
-            {% else %}
-              <a class="mobile-title" href="{{ url_for('record.metadata', recid=record['control_number']) }}">
-                {{ record['titles[0].title']|capitalize }}
-              </a>
-            {% endif %}
-          </b>
-      </h4>
-      {% if record.authors %}
-        <div class="authors">
-          {{ render_record_authors(is_brief=true, show_affiliations=false) }}
-        </div>
-      {% endif %}
-      {% if record.get('publication_info') or record.get('dois')%}
+      <div class="panel-body" >
         <div class="row">
-          {% if record.get('publication_info') %}
-            <div class="col-md-6 ">
-              {{ record_journal_info() }}
+          <div class="col-md-9" id="left-column">
```

Comment:
This is a problem as many elements in the page have the same id `left-column`. If we need an id then you can add dynamically a suffix with the record id. Otherwise a class can be used?